packages/core/src/kernelCoreTemplate.js

```diff
@@ -73,6 +73,7 @@
     const { getEndowmentsForConfig, makeMinimalViewOfRef, applyEndowmentPropDescTransforms } = templateRequire('makeGetEndowmentsForConfig')(generalUtils)
     const { prepareCompartmentGlobalFromConfig } = templateRequire('makePrepareRealmGlobalFromConfig')(generalUtils)
     const { strictScopeTerminator } = templateRequire('strict-scope-terminator')
+    const SNOW = globalRef.SNOW || ((cb, win) => cb(win))
```

Comment:
this fallback seems like its a "fail to unsafe" situation which is usually not what we want

Reply:
21a7787a6